userspace/libsinsp/cri.hpp

```diff
@@ -695,6 +681,9 @@ bool cri_interface<api>::parse_containerd(const typename api::ContainerStatusRes
 	if(root.isMember("sandboxID") && root["sandboxID"].isString())
 	{
 		const auto pod_sandbox_id = root["sandboxID"].asString();
+		// Add the pod sandbox id as label to the container.
+		// This labels is needed by the filterchecks code to get the pod labels.
+		container.m_labels["io.kubernetes.sandbox.id"] = pod_sandbox_id;
```

Comment:
This is nice and an improvement, ty!! We can further refine and reorganize things in subsequent PRs.